src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1745,32 +1728,6 @@ because xbuild doesn't support framework reference assemblies.
   </PropertyGroup>
 </Target>
 
-<Target Name="_CleanupOldStaticResources"
-    Condition=" Exists ('$(_AndroidIntermediateJavaSourceDirectory)mono\android\app\NotifyTimeZoneChanges.java') ">
-  <ItemGroup>
-    <_OldStaticResources Include="$(_AndroidIntermediateJavaSourceDirectory)mono\android\incrementaldeployment\MultiDexLoader.java" />
```

Comment:
How many released did we switch over to the new pre built runtime? I think it should be safe to remove this code if we have had a couple of released between then and now

Reply:
I think the new clean behavior will delete these files now 👍 

We still have a test checking for these files getting deleted: https://github.com/xamarin/xamarin-android/blob/2fd73856bde0b0800b2455485cf15e6b4b393bec/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs#L3852-L3899